tonic/meta.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: tonic
+  version: "0.13.1"
+
+source:
+  fn: tonic-0.13.1.tar.gz
+  url: https://pypi.python.org/packages/source/p/tonic/tonic-0.13.1.tar.gz
```

Comment:
I'm fairly certain this isn't the right url.  I don't have any memory of adding `tonic` to pypi. Can we checkout from github?